node/test/toolrunnertests.ts

```diff
@@ -34,15 +36,15 @@ describe('Toolrunner Tests', function () {
 
         tl.pushd(__dirname);
 
-        var _testExecOptions: trm.IExecOptions = {
+        var _testExecOptions = <trm.IExecOptions>{
```

Comment:
I added another field to the interface, which causes compile errors. Rather than explicitly specify the field everywhere this object is constructed, I opted to just chang how the object is instantiated so we don't need to worry about this next time a field is added.